recipes-overlayed/kselftests/kselftests_4.12-rc7.bb

```diff
@@ -47,6 +49,7 @@ PACKAGE_BEFORE_PN = " \
 	${PN}-bpf \
 	${PN}-breakpoints \
 	${PN}-capabilities \
+	${PN}-cpufreq \
```

Comment:
no, ipc isn't missing, it's arch specific so it isn't in the the generic list but below.